station/app/crud/crud_docker_trains.py

```diff
@@ -66,15 +66,20 @@ def get_trains_by_active_status(self, db: Session, active=True, limit: int = 0)
             trains = db.query(DockerTrain).filter(DockerTrain.is_active == active).all()
         return trains
 
-    # todo maybe remove for new solution by tyra
-    def add_if_not_exists(self, db: Session, train_id: str, created_at: str = None):
+    def add_if_not_exists(self, db: Session, train_id: str, created_at: str = None, updated_at: str = None):
```

Comment:
updated_at, created_at should reflect local state -> created_at = datetime.now() updated_at= None